### PR TITLE
[ServiceBus] Add identity, encryption cmdlets and edited namespace create cmdlet to include MSI properties.

### PR DESCRIPTION
**Pull request by damodaravadhani**
**Description**
Adding MSI cmdlets. (az servicebus namespace identity)
Adding MSI encryption cmdlets (az servicebus namespace encryption)
Edited (az servicebus namespace create) to include MSI properties

**Testing Guide**
***Create namespace with user assigned identity encryption***

`az servicebus namespace create --resource-group {resourcegroupname} --name {namespacename} --sku premium --mi-user-assigned {resourceIdOfMSI1} {resourceIdOfMSI2} --encryption-config key-name=key1 key-vault-uri=https://{uri} user-assigned-identity=/subscriptions/{subscriptionid}/resourceGroups/{resourcegroupname}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{name} --encryption-config key-name={keyname} key-vault-uri={keyvaulturi} user-assigned-identity=/subscriptions/{subscriptionid}/resourceGroups/{resourcegroupname}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{name}`

***Add encryption configs to created namespace***

`az servicebus namespace encryption add --resource-group damorg --namespace-name testCLIMSISdkRound1 --encryption-config key-name=key1 key-vault-uri=https://{name}.vault.azure.net user-assigned-identity=/subscriptions/{subscriptionid}/resourceGroups/{resourcegroupname}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{name} --encryption-config key-name=key2 key-vault-uri=https://{name}.vault.azure.net user-assigned-identity=/subscriptions/{subscriptionid}/resourceGroups/{resourcegroupname}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{name}`

***Remove Encryption Configs from created namespace***

`az eventhubs namespace encryption remove --encryption-config key-name=key1 key-vault-uri=https://{name}.vault.azure.net user-assigned-identity=/subscriptions/{subscriptionid}/resourceGroups/{resourcegroupname}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{name}`

***Add system assigned and user assigned identity to namespace***

`az servicebus namespace identity assign --resource-group {rgname} --namespace-name {name} --system-assigned --user-assigned /subscriptions/{subscriptionid}/resourceGroups/{resourcegroupname}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{name}` 

***Remove system assigned and user assigned identity from namespace***

`az servicebus namespace identity remove --resource-group {rgname} --namespace-name {name} --system-assigned --user-assigned /subscriptions/{subscriptionid}/resourceGroups/{resourcegroupname}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{name}`

**History Notes**

[EventHub] `az servicebus namespace create`: Add  `--user-assigned`, `--system-assigned`, `--encryption-config`
[EventHub] `az servicebus namespace identity`: Cmdlets for event hubs identity
[EventHub] `az servicebus namespace encryption`: Cmdlets for event hubs encryption

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [X] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by damodaravadhani** [comment]
Could you please add the February 2022 (2022-03-01) label to this PR ? We want this to go in the February release. Thanks

File: src/azure-cli/azure/cli/command_modules/servicebus/_help.py
Comment (wangzelin007) [inline src/azure-cli/azure/cli/command_modules/servicebus/_help.py:656]:
```suggestion
short-summary: Assign System or User or System, User assigned identities to a namespace
```

File: src/azure-cli/azure/cli/command_modules/servicebus/_help.py
Comment (wangzelin007) [inline src/azure-cli/azure/cli/command_modules/servicebus/_help.py:665]:
```suggestion
short-summary: Removes System or User or System, User assigned identities from a namespace
```

**Review by wangzelin007 — APPROVED** [review]

**Comment by damodaravadhani** [comment]
Could we merge this PR if there aren't any additional concerns ? Thank you.

**Comment by evelyn-ys** [comment]
@jsntcy @zhoxing-ms This PR still needs your review, can you help?

**Review by zhoxing-ms — APPROVED** [review]

**Comment by damodaravadhani** [comment]
Hey @evelyn-ys , looks like we have enough approvals, could we go ahead and merge this. Thank you.

**Pull request merged**
